lib/Doctrine/Migrations/Metadata/Storage/TableMetadataStorage.php

```diff
@@ -123,7 +127,7 @@ public function complete(ExecutionResult $result) : void
         } else {
             $this->connection->insert($this->configuration->getTableName(), [
                 $this->configuration->getVersionColumnName() => (string) $result->getVersion(),
-                $this->configuration->getExecutedAtColumnName() => $result->getExecutedAt() ? $result->getExecutedAt()->format($this->platform->getDateTimeFormatString()): null,
+                $this->configuration->getExecutedAtColumnName() => $result->getExecutedAt()!== null ? $result->getExecutedAt()->format($this->platform->getDateTimeFormatString()): null,
```

Comment:
this could be simplified by using the `$types` third argument instead, and letting DBAL perform the conversion